client/components/plans/plan-icon/index.jsx

```diff
@@ -0,0 +1,497 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PLAN_FREE,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_JETPACK_FREE,
+	PLAN_JETPACK_BUSINESS,
+	PLAN_JETPACK_BUSINESS_MONTHLY,
+	PLAN_JETPACK_PREMIUM,
+	PLAN_JETPACK_PREMIUM_MONTHLY,
+	PLAN_PERSONAL
+} from 'lib/plans/constants';
+
+export default class PlanIcon extends Component {
+	render() {
+		const { plan } = this.props;
+
+		return (
+			<div className="plan-icon">
```

Comment:
Let's set some default width/height on the plan-icon class, but also pass through any classes on the className property so we can decorate it with more class names if needed, eg:
```javascript
render() {
    const { plan, className } = this.props;
    const classes = classNames( 'plan-icon', className );
    return (
        <div className={ classes }>
          //...
        </div>
    );
}
```